integration-tests/run-intg-test.py

```diff
@@ -405,25 +424,26 @@ def build_module(module_path):
     """
     logger.info('Start building a module. Module: ' + str(module_path))
     if sys.platform.startswith('win'):
-        subprocess.call(['mvn', 'clean', 'install', '-B',
+        subprocess.call(['mvn', 'clean', 'install', '-fae', '-B',
                          '-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'],
                         shell=True, cwd=module_path)
     else:
-        subprocess.call(['mvn', 'clean', 'install', '-B',
+        subprocess.call(['mvn', 'clean', 'install', '-fae', '-B',
                          '-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'],
                         cwd=module_path)
     logger.info('Module build is completed. Module: ' + str(module_path))
 
 
 def save_log_files():
-    log_storage = Path(workspace + "/" + LOG_STORAGE)
-    if not Path.exists(log_storage):
-        Path(log_storage).mkdir(parents=True, exist_ok=True)
-    log_file_paths = LOG_FILE_PATHS[product_id]
-    if log_file_paths:
-        for file in log_file_paths:
+    log_folder = Path(workspace + "/" + LOG_STORAGE)
+    if Path.exists(log_folder):
+        shutil.rmtree(log_folder)
+    log_file_paths = ARTIFACT_REPORTS_PATHS[product_id]
+    for key, value in log_file_paths.items():
+        for file in value:
             absolute_file_path = Path(workspace + "/" + product_id + "/" + file)
             if Path.exists(absolute_file_path):
+                log_storage = Path(workspace + "/" + LOG_STORAGE + "/" + key)
```

Comment:
Above comment applies to the `LOG_STORAGE`.We can make a common directory for all outputs like `test_outputs` and save all the log files and surefire reports under that directory.

Reply:
Fixed. Please refer latest commits